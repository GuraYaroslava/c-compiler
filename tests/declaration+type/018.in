int foo(int);

foo(1);