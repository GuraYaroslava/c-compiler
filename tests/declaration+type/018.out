table 0:
`int`
`float`
`char`

table 1:
variable `foo` type of function(variable `abstract-0` type of `int`) returning `int`

tree 0:
funtion: 
       foo
args: 
              1


